.github/workflows/linkcheck.yml

```diff
@@ -0,0 +1,15 @@
+name: Link-check
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run linksafe
+        uses: TechWiz-3/linksafe@main
+        with:
+          dirs: ".,./docs"
+          verbose: true
+          whitelist_files: "./CONTRIBUTING.md"
+          whitelist_links: "https://ceph.io/,https://mattermost.com/,https://tyk.io/,https://konghq.com/,https://konghq.com/products/kong-gateway/kong-proxy,https://www.saltstack.com/,https://www.docker.com/,https://www.gluster.org/,https://cloud.vmware.com/,https://www.vultr.com/,https://www.linkedin.com/in/wmariuss"
```

Comment:
Why is this list? Should not scan all?

Reply:
As the link checker uses HTTP response codes to determine whether a link is good or not, some links will come with a status code that *could* be broken but could also be OK. When this happens and you know a link is good, you don't want the checker flagging them in future, so you add them to the whitelist